### PR TITLE
chore: rename switch to toggle

### PR DESCRIPTION
**Pull request by tanvibhakta**
## Description

In the effort to have well named components, we are merging the Switch component into the Toggle component. 
Fixes https://github.com/appsmithorg/appsmith/issues/18822
Depends on https://github.com/appsmithorg/design-system/pull/282

## Type of change
Chore

## How Has This Been Tested?
- Jest
- Cypress

### Test Plan
Regression on both switch and toggle 
Switch usage: https://docs.google.com/spreadsheets/d/1np7jQdiQa0nyryOBnNa927NkGDplG9M2gb7qnoZIIyM/edit#gid=1676845816
Toggle Usage: https://docs.google.com/spreadsheets/d/1np7jQdiQa0nyryOBnNa927NkGDplG9M2gb7qnoZIIyM/edit#gid=943732746

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by albinAppsmith — DISMISSED** [review]

**Review by albinAppsmith — APPROVED** [review]

File: app/client/src/components/formControls/SwitchControl.tsx [hunk 1]
```diff
@@ -56,13 +56,12 @@ export class SwitchField extends React.Component<SwitchFieldProps, any> {
   render() {
     return (
       <SwitchWrapped data-cy={this.props.input.name}>
```
Comment (rohitagarwal88) [inline app/client/src/components/formControls/SwitchControl.tsx:58]: @tanvibhakta : Can we rename SwitchWrapped to ToggleWrapped?

Comment (tanvibhakta) [reply app/client/src/components/formControls/SwitchControl.tsx:58]: This is from the file SwitchControl, where there are other components/type/variants with the prefix `Switch`. I wouldn't want to rename only one component in this file but I thought renaming the whole file (and all it's usages) might be out of the scope of this issue. 
I'd prefer we leave it as it is or rename everything. What do you think?

**Review by albinAppsmith — APPROVED** [review]

**Comment by tanvibhakta** [comment]
This PR will not be merged because we are picking up refactoring instead.

**Pull request closed without merging**